pkg/provisioner/ironic/clients/client.go

```diff
@@ -93,8 +93,8 @@ func IronicClient(ironicEndpoint string, auth AuthConfig, tls TLSConfig) (client
 
 	// Ensure we have a microversion high enough to get the features
 	// we need. Update docs/configuration.md when updating the version.
-	// Version 1.81 allows retrival of Node inventory
-	client.Microversion = "1.81"
+	// Version 1.86 allows using Firmware Interface
+	client.Microversion = "1.86"
```

Comment:
This will need to be reverted before merging - see https://github.com/metal3-io/baremetal-operator/pull/1435

Reply:
yeah, I will update when we have #1435 merged